docs/logstash_migration_guide.md

```diff
@@ -0,0 +1,27 @@
+# Logstash Migration Guide
+
+This guide describes running Data Prepper with a Logstash configuration.
+
+As mentioned in the [Getting Started](getting_started.md) guide, to configure Data Prepper, you require the following 
+two files:
+* `data-prepper-config.yaml`
```

Comment:
Since we are providing a default `data-prepper-config` for Docker in 1.2, we should simplify this and only discuss the `pipeline.yaml` to `logstash.conf`